utils/src/parallel_restore.c

```diff
@@ -1849,7 +1899,7 @@ static spr_err_t scoutfs_check_if_previous_mount(int fd)
 	for (i = 0; i <  SCOUTFS_QUORUM_MAX_SLOTS; i++) {
 		blkno = SCOUTFS_QUORUM_BLKNO + i;
 		err = read_block(fd, blkno, SCOUTFS_BLOCK_SM_SHIFT, (void **)&blk);
-		if (blk == NULL;)
+		if (blk == NULL)
```

Comment:
Don't fix up unrelated bugs without any comments like this. Make them a separate commit.